tools/release.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
how about env bash instead? bash lives elsewhere on FreeBSD and some other systems.